pkg/scheduler/api/resource_info.go

```diff
@@ -567,3 +577,29 @@ func ParseResourceList(m map[string]string) (v1.ResourceList, error) {
 	}
 	return rl, nil
 }
+
+func GetMinResource() float64 {
+	return minResource
+}
+
+// ResourceNames struct defines resource name collection
+type ResourceNames struct {
+	Names []v1.ResourceName
+}
+
+// Subset judges whether dimensions in r is subset of that of rr
```

Comment:
// Subset judges whether dimensions in r is subset of that of rr
->
// Subset judges whether r is part of rr